tests/rptest/wasm/wasm_test.py

```diff
@@ -104,6 +106,26 @@ def to_ots(src):
     def _verify_materialized_outputs(outputs):
         return all([is_materialized_topic(t) for t in outputs.keys()])
 
+    def _create_input_producers(self, topics, rate_lookup):
+        producers = {}
+        for tp_spec in topics:
+            topic = tp_spec.name
+            num_records = rate_lookup[topic]
+            self.logger.info(
+                f"Input producer assigned: {tp_spec} - {num_records}")
+
+            producer = VerifiableProducer(self._test_context,
```

Comment:
In future I think we wanna use `FranzGoVerifiableProducer/Counsumer` instead of this
@jcsp should we create meta-issue for si-verifier functionality

Reply:
In future yes, but probably not right now.  I'd like to be a bit gradual about rolling it out, as we still need to make some improvements to the interface (JSON output for message counts etc).